packages/framework/src/Model/Payment/PaymentDataFactory.php

```diff
@@ -29,22 +31,30 @@ class PaymentDataFactory implements PaymentDataFactoryInterface
      */
     protected $imageFacade;
 
+    /**
+     * @var \Shopsys\FrameworkBundle\Model\Pricing\Currency\CurrencyFacade
+     */
+    protected $currencyFacade;
+
     /**
      * @param \Shopsys\FrameworkBundle\Model\Payment\PaymentFacade $paymentFacade
      * @param \Shopsys\FrameworkBundle\Model\Pricing\Vat\VatFacade $vatFacade
      * @param \Shopsys\FrameworkBundle\Component\Domain\Domain $domain
-     * @param \Shopsys\FrameworkBundle\Component\Image\ImageFacade|null $imageFacade
+     * @param \Shopsys\FrameworkBundle\Component\Image\ImageFacade $imageFacade
+     * @param \Shopsys\FrameworkBundle\Model\Pricing\Currency\CurrencyFacade $currencyFacade
      */
     public function __construct(
         PaymentFacade $paymentFacade,
         VatFacade $vatFacade,
         Domain $domain,
-        ?ImageFacade $imageFacade = null
+        ImageFacade $imageFacade,
```

Comment:
How can I get null srvice from DC?

Reply:
It is done that way because extension from project-base would lead to error - so it is BC break. We will have US for removing deprecations in the future.

Reply:
Ok, I got it back